ipa-core/src/protocol/basics/mod.rs

```diff
@@ -3,6 +3,8 @@ mod if_else;
 pub(crate) mod mul;
 mod reshare;
 mod reveal;
+#[allow(dead_code)]
+mod shard_fin;
```

Comment:
the `protocol/hybrid/mod.rs` file is ready to have this included.

Reply:
clippy will still yell at me, so I'll enable it in a follow up